src/components/autofill.ts

```diff
@@ -3,7 +3,7 @@ import { clip } from "../helpers/misc";
 import { SpreadsheetEnv, Viewport } from "../types";
 import { startDnd } from "./helpers/drag_and_drop";
 
-const { Component } = owl;
+const Component = owl.Component;
```

Comment:
I need to look into it more thoroughly - it has been solved when owl introduced a global `owl.d.ts` file. I isolted the commit since it won't be necessary for the next versions :)